sample_apps/simple_app_under_oe/enclave/ecalls.cc

```diff
@@ -192,8 +192,8 @@ void clear_sensitive_data() {
 bool cold_init() {
   return app_trust_data->cold_init(public_key_alg,
                                    symmetric_key_alg,
-                                   initialized_cert_size,
                                    initialized_cert,
+                                   initialized_cert_size,
```

Comment:
HI, @yelvmw -- Thanks for catching this.

I re-tested this change on SGX box for both `simple_app_under_gramine` and `simple_app_under_oe`.

Now, with this fix, the latter one also compiles and runs cleanly.